### PR TITLE
enabling nginx to listen on ipv6

### PR DESCRIPTION
**Pull request by venturel**
Please verify the changes in the config file to enable ipv6 for nginx that currently is not enabled at least on irail.be

### CODE REVIEW

**Comment by duboisph** [comment]
👍

**Comment by duboisph** [comment]
Good idea, working on it. Merged master back to this branch.

**Comment by SamClinckspoor** [comment]
👍

**Comment by venturel** [comment]
I'd honestly enable ipv6 by default

**Comment by duboisph** [comment]
Enabling it by default might break current customers, where the redirect and static templates are used in combination with a specific IP.

So unless we go over every customer's code, I wouldn't enable it by default.

**Comment by venturel** [comment]
so perhaps disable it by default only on redirect and static templates.

**Comment by duboisph** [comment]
Ok, that's something we can do and should be safe. What does the rest think?

**Comment by duboisph** [comment]
Going to merge with default disabled now, while working on a better solution to enable IPv6 by default.

File: templates/etc/nginx/sites-available/static.conf.erb [hunk 1]
```diff
@@ -3,7 +3,12 @@
 server {
 
   # Listen
+  <% if @enable_ipv6 %>
+  listen <%= @port %>;
```
Comment (duboisph) [inline templates/etc/nginx/sites-available/static.conf.erb]: Not relevant anymore :P

**Pull request merged**